src/icons/general/IconAdd.js

```diff
@@ -12,20 +12,19 @@ type Props = {
     width?: number,
 };
 
-const IconAdd = ({ className = '', color = '#000000', height = 24, title, width = 24 }: Props) => (
+const IconAdd = ({ className = '', color = '#222', height = 16, title, width = 16 }: Props) => (
```

Comment:
Ideally, the default `color` value should come from our `variables` module and use one of the new `bdlNeutral` colors. Same comment for the new hidden icon.